<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TaoThuNgan</class>
 <widget class="QMainWindow" name="TaoThuNgan">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>348</width>
    <height>638</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tạo Tài Khoản Thu Ngân</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#Form
{
	background-image: url(:/background/BACKGROUND/bg1.jpg);
}</string>
  </property>
  <widget class="QWidget" name="Form">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>291</width>
      <height>531</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame
{
	background-color: #222;
	border-radius: 20px
}

QPushButton
{
	background: red;
	border-radius: 10px;
}

QLineEdit
{
	background: transparent;
	border:none;
	color:#717072;
	border-bottom: 1px solid #717072;
}

QLabel
{
	color: white;
	font: 900 18pt &quot;Segoe UI&quot;;
}

QGroupBox
{
	border: 2px solid rgb(255, 255, 127);
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>114</y>
       <width>261</width>
       <height>401</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <widget class="QLabel" name="ID_LABEL">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>91</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>18</pointsize>
        <italic>false</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>User ID</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="PW_LABEL">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>194</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>18</pointsize>
        <italic>false</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Password</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="ID_LINE">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>49</y>
        <width>241</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
      <property name="placeholderText">
       <string>User ID</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="PW_LINE">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>234</y>
        <width>241</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
      <property name="placeholderText">
       <string>Password</string>
      </property>
     </widget>
     <widget class="QPushButton" name="CA_BUTTON">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>290</y>
        <width>241</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>CREATE ACCOUNT</string>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="NAME_LABEL">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>101</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>18</pointsize>
        <italic>false</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Name</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="NAME_LINE">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>140</y>
        <width>241</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Normal</enum>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
      <property name="placeholderText">
       <string>Enter Your Name</string>
      </property>
     </widget>
     <widget class="QPushButton" name="EXIT_BUTTON">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>345</y>
        <width>241</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 85, 127);</string>
      </property>
      <property name="text">
       <string>GO BACK</string>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>66</y>
       <width>291</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>20</pointsize>
       <italic>false</italic>
       <bold>true</bold>
       <stylestrategy>PreferDefault</stylestrategy>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
font: 900 20pt &quot;Segoe UI&quot;;</string>
     </property>
     <property name="text">
      <string>CREATE ACCOUNT</string>
     </property>
     <property name="textFormat">
      <enum>Qt::PlainText</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QToolButton" name="toolButton">
    <property name="geometry">
     <rect>
      <x>115</x>
      <y>20</y>
      <width>121</width>
      <height>121</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QToolButton
{
	background-color: qconicalgradient(cx:0.489, cy:0.5, angle:0, stop:0.460227 rgba(246, 230, 116, 255));
	border-radius: 60px;
}
</string>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="icon">
     <iconset resource="ing.qrc">
      <normaloff>:/icon/ICON/written-paper.png</normaloff>:/icon/ICON/written-paper.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>80</width>
      <height>80</height>
     </size>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="ing.qrc"/>
 </resources>
 <connections/>
</ui>
